src/app/core/dspace-rest-v2/dspace.serializer.ts

```diff
@@ -1,19 +1,16 @@
-import { Serialize, Deserialize } from 'cerialize';
+import { Deserialize, Serialize } from 'cerialize';
 
 import { Serializer } from '../serializer';
-import { DSpaceRESTV2Response } from './dspace-rest-v2-response.model';
-import { DSpaceRESTv2Validator } from './dspace-rest-v2.validator';
 import { GenericConstructor } from '../shared/generic-constructor';
-import { hasNoValue, hasValue } from '../../shared/empty.util';
 
 /**
  * This Serializer turns responses from v2 of DSpace's REST API
  * to models and vice versa
  */
-export class DSpaceRESTv2Serializer<T> implements Serializer<T> {
+export class DSpaceSerializer<T> implements Serializer<T> {
```

Comment:
I named it differently because I used it in parallel with the old DSpaceRESTv2Serializer for a while. It still returned normalized objects while this serializer returns domain models. After I got rid of the all the normalized models, I deleted the old serializer, and git registered it as a rename.

That said, it _is_ still applicable to the v2 of the dspace rest api (the v1 being the dspace 5 and 6 one) and when we started with dspace 7 we agreed to make a v3 if we add breaking changes after the release of dspace 7 stable. (I imagine dspace 8 will likely do that) so that people can run v2 and v3 at the same time if they have apps that depend on the old way of working.

But you could argue we should only create folders for different rest versions once we have them, and I can agree with that. But since it's already there, we might as well leave it.

Reply:
This is fine for now. I had forgotten why we were calling this `v2`  but you are correct.  A part of me wonders if it's better to just version the REST API alongside the version of DSpace (i.e. jump this to v7 REST API).  But, that's not part of this PR :)  We can simply leave this as-is for now.